### ISSUE TITLE
love: symbol lookup error: /usr/lib/liblove-11.4.so: undefined symbol: mpg123_open_handle_64

### ISSUE THREAD

**Issue by mxmilkiib** [love2d/love#1972]
https://github.com/rgrams/multiviewer-2/issues/6#issuecomment-1737667922

On running `love` I get;
`love: symbol lookup error: /usr/lib/liblove-11.4.so: undefined symbol: mpg123_open_handle_64`

I've tried both the official Arch love package and the AUR love-git, also mpg123-svg from the AUR over the official mpg123 binary package.

**Comment by slime73** [comment]
Without digging into it too much this looks a lot like a third-party packaging issue to me. Probably either the mpg123 package you have is a different version than what love needs and whoever set up the love arch package didn't tag it, or the mpg123 package has been misconfigured in general to not export all the symbols it should?

**Comment by Tekkitslime** [comment]
nm -D /usr/lib/libmpg123.so doesnt seem to have that symbol, so yeah, arch issue. AUR love-git package works though.

**Comment by MikuAuahDark** [comment]
Since this is ArchLinux packaing issue, I don't think there's much we can do. ArchLinux bug report is here: https://bugs.archlinux.org/task/79801?project=1&string=love

Alternatively, use the AppImage for time being.

**Comment by gtroja** [comment]
not related to the subject, but I wanna thanks @Tekkitslime, he helped me solve an issue in running gta III via re3+portmaster in the stock os of arbenic rg353v. The logs were complaing about  `undefined symbol: mpg123_open`, I searched for libmpg123.so and runned the `nm` command (which I didn't know existed) and sure enough, mpg123_open wasn`t listed. Since I had a ARK OS in another card, I searched for the lib there and that version had the symbol. I simply replaced the stock for the arkOS and the game ran! thx again!
